embedded-graphics/src/primitives/triangle.rs

```diff
@@ -406,6 +452,27 @@ mod tests {
         assert_eq!(tri.next(), None);
     }
 
+    #[test]
+    fn it_draws_filled_strokeless_tri() {
+        let mut tri = Triangle::new(Point::new(2, 2), Point::new(2, 4), Point::new(2, 4))
+            .into_styled(
+                PrimitiveStyleBuilder::new()
+                    .fill_color(BinaryColor::On)
+                    .build(),
+            )
+            .into_iter();
+
+        // Nodes are returned twice. first line a and b yield the same point.
+        // After that line a ends where line c starts.
+        assert_eq!(tri.next(), Some(Pixel(Point::new(2, 2), BinaryColor::On)));
+        assert_eq!(tri.next(), Some(Pixel(Point::new(2, 2), BinaryColor::On)));
+        assert_eq!(tri.next(), Some(Pixel(Point::new(2, 3), BinaryColor::On)));
+        assert_eq!(tri.next(), Some(Pixel(Point::new(2, 3), BinaryColor::On)));
+        assert_eq!(tri.next(), Some(Pixel(Point::new(2, 4), BinaryColor::On)));
+        assert_eq!(tri.next(), Some(Pixel(Point::new(2, 4), BinaryColor::On)));
+        assert_eq!(tri.next(), None);
```

Comment:
Does this test (and the comment) make sense for the fill? Maybe a `MockDisplay` test would be better suited.

Reply:
Changed to a MockDisplay test. The other tests like this in the file can be migrated some other time :)